content/docs/usage/certificaterequest.md

```diff
@@ -6,6 +6,10 @@ description: 'cert-manager core concepts: CertificateRequests'
 > **apiVersion:** cert-manager.io/v1  
 > **kind:** CertificateRequest
 
+<div style={{textAlign: "center"}}>
+<object data="/images/request-certificate-overview/request-certificate-certificaterequest.svg"></object>
+</div>
```

Comment:
I'm unsure about these diagrams. Unless the reader is comfortable with all the terms (CSI driver, etc), the diagram doesn't really help understanding. What idea are you trying to convey with these diagrams?

Reply:
IMO, this diagram should only show the "resources at play": the Gateway (with the principal annotation that enables the workflow) + the CertificateResource + the Secret. Anything else will confuse people.

Reply:
It aims to show you what resources are created by the user & what resources are auto-generated for each of the different paths. (eg. If you annotate an Ingress, that will create a Certificate resource.